frontend/src/components/Tool/Tools.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import {
+    Card, CardBody, CardTitle, Row, Col
+  } from 'reactstrap'
+    
+import '../../style/Tools.css'
+import reactLogo from './Tool_logos/React_logo.jpg'
+import nodeLogo from './Tool_logos/NodeJS_logo.png'
+import jiraLogo from './Tool_logos/Jira_logo.jpg'
+import postmanLogo from './Tool_logos/Postman_logo.jpg'
+import awsLogo from './Tool_logos/AWS_logo.jpg'
+import herokuLogo from './Tool_logos/Heroku_logo.jpg'
```

Comment:
Lets move the images into their own folder. Under src create a new folder named images and put these images in that folder.

Reply:
I'm able to do all that but I get this error when I just try to import the images folder into Tools.js.

![Images Error](https://user-images.githubusercontent.com/43736409/112774358-0fd54680-8fff-11eb-9643-ece371d1900a.png)

The code I import is just "import '../images'"

Alternatively putting it in like this:
"img src="../images/React_logo.jpg" alt="Card image cap" className="centerlogo"
It just shows the alt text and this occurs for all the images. I think I originally tried to do this but the way I found that worked was what I originally did. This seems really simple but I don't know whats happening